src/app/pages/directory-service/components/active-directory/active-directory.component.ts

```diff
@@ -404,6 +404,8 @@ export class ActiveDirectoryComponent implements FormConfiguration {
       } else {
         data.netbiosalias = [];
       }
```

Comment:
Could replace this block with
`
if(!data.netbiosalias?.trim().length) { data.netbiosalias = []; }
else { data.netbiosalias = data.netbiosalias.trim().split(' '); }
`

Reply:
Not too important, the form has ben refactored in Bluefin.